_functions.sh

```diff
@@ -592,6 +589,32 @@ initialize_product_settings() {
         env_var "INSTANCE_DESCRIPTION" "${PRODUCT_DESCRIPTION} ${PRODUCT_VERSION} (${INSTANCE_ID})"
       fi
 
+      # Versions specific configuration
+      if [[ "${PRODUCT_BRANCH}" =~ ^(5.0) ]]; then
```

Comment:
you should not use an if statement but a case statement because we need to fail when PLF 5.1 branch will be introduced of it will go back to the old driver download behavior

Reply:
I will improve the comment but I plan to also add the DEPLOYMENT_APPSRV_VERSION computation here

Reply:
Test updated. I also add a test to limit this to plf deployment because other deployment type like buypage were broken